homeassistant/components/huawei_lte/__init__.py

```diff
@@ -298,7 +298,7 @@ def cleanup(self, *_: Any) -> None:
 class HuaweiLteData:
     """Shared state."""
 
-    hass_config: dict = attr.ib()
+    hass_config: ConfigType = attr.ib()
     # Our YAML config, keyed by router URL
     config: dict[str, dict[str, Any]] = attr.ib()
```

Comment:
By the description it looks like this should be:
```
config: dict[str, ConfigType] = attr.ib()
```

or `config: ConfigType = attr.ib()`

Reply:
I think it's correct as is, see how it's constructed in `async_setup`. (The former suggestion would "fit", but that'd be just a coincidence, this data isn't really defined in terms of `ConfigType`.)